fragments/rgw/test_versioning_objects_enable.yaml

```diff
@@ -0,0 +1,10 @@
+tests:
```

Comment:
We could further structure the folder to `rgw/versioning/test_objects_enable.yaml`